### ISSUE TITLE
Cannot Connect to node

### ISSUE THREAD

**Issue by cyborg-637** [bpanel-org/bpanel#99]
When I attempt to ~/bpanel$ npm run start:poll, I get the following error

> @bpanel/bpanel@0.3.0-alpha start:poll /home/guthrie/bpanel
> node --max_old_space_size=4096 server --dev --watch-poll

internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module 'bfile'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:2Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.5 LTS
Release:	16.04
Codename:	xenial
5)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/home/guthrie/bpanel/server/index.js:8:12)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @bpanel/bpanel@0.3.0-alpha start:poll: `node --max_old_space_size=4096 server --dev --watch-poll`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @bpanel/bpanel@0.3.0-alpha start:poll script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/guthrie/.npm/_logs/2018-09-14T00_50_00_524Z-debug.log

nodejs -v
v10.10.0

Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.5 LTS
Release:	16.04
Codename:	xenial

nodejs -v
v10.10.0
npm -version
6.4.1

**Comment by bucko13** [comment]
Hi @cyborg-637 ,
I’m so sorry it took so long to get back to you on this. Not sure why I never saw a notification about this. bPanel uses `bfile` as a drop in for `fs` as it adds extra functionality and compatibility. The error you were getting was we did a simple search and replace to add that in and it got included in the preinstall script, which obviously wouldn’t work since the module hadn’t been installed yet. 

Long story short, this should be fixed now! That script now uses `fs` during the setup (make sure you’ve also run `npm install` if you’ve pulled a new version of the repo down). Thereve been a lot of updates to the platform since you originally posted the issue so hopefully you’ll give bPanel another try!
